views/restaurants.html

```diff
@@ -0,0 +1,479 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Restaurants</title>
+    <link rel="stylesheet" href="../public/css/restaurants.css">
+</head>
+<body>
+    <nav class="navbar navbar-fixed flex flex-wrap flex-between  navbar-bg nav-zindex">
+        <div class="flex flex-space-evenly flex-align-item-center ">
+            <ul class=" text-deocration-none flex flex-wrap flex-align-item-center gap">
+                <li class="pb-1 nav-text navbar-white-text flex flex-align-item-center gap-1 hide">
+                    <span class="material-icons snackbar-icons  navbar-icons">menu</span>
+                </li>
+                <a href="../views/home.html" class="text-deocration-none list-style-none">
+                    <li class="pb-1  navbar-white-text flex flex-align-item-center gap-1">
+                        <span class="font-bold label-text label-text-primary">Zeplin</span>
+                    </li>
+                </a>
+            </ul>
+        </div>
+        <div class="flex flex-space-evenly flex-align-item-center ">
+            <ul class=" text-deocration-none flex flex-wrap gap-col-2">
+
+                <a href="../views/home.html" class="text-deocration-none">
+                    <li class="pb-1 nav-text  navbar-white-text flex flex-align-item-center gap-1">
+                        <span class="material-icons label-text label-text-primary">home</span>
+                        <span class="label-text label-text-primary">Home</span>
+                    </li>
+                </a>
+
+                <a href="../views/restaurants.html" class="text-deocration-none">
+                    <li class="pb-1 nav-text  navbar-white-text flex flex-align-item-center gap-1 ">
+                        <span class="material-icons label-text label-text-primary">dinner_dining</span>
+                        <span class="label-text label-text-primary">Restaurants</span>
+                    </li>
+                </a>
+                <a href="../views/cart.html" class="text-deocration-none">
+                    <li class="pb-1 nav-text  navbar-white-text flex flex-align-item-center gap-1">
+                        <span class="material-icons label-text label-text-primary">person</span>
+                        <span class="label-text label-text-primary">cart</span>
+                    </li>
+                </a>
+                <a href="../views/wishlist.html" class="text-deocration-none">
+                    <li class="pb-1 nav-text  navbar-white-text flex flex-align-item-center gap-1">
+                        <span class="material-icons label-text label-text-primary">queue</span>
+                        <span class="label-text label-text-primary">wishlisht</span>
+                    </li>
+                </a>
+                <a href="../views/login.html" class="text-deocration-none">
```

Comment:
in global css 

a {
text-deocration : none }

because in a tag we  rarely  need it

Reply:
you do not need to use that class again & again